sql/analyzer/rules.go

```diff
@@ -104,17 +104,26 @@ func qualifyColumns(a *Analyzer, n sql.Node) (sql.Node, error) {
 }
 
 func resolveDatabase(a *Analyzer, n sql.Node) (sql.Node, error) {
-	_, ok := n.(*plan.ShowTables)
-	if !ok {
-		return n, nil
-	}
+	// TODO Database should implement node,
+	// and ShowTables and CreateTable nodes should be binaryNodes
+	switch v := n.(type) {
+	case *plan.ShowTables:
```

Comment:
Can we use `fallthrough` so we don't repeat the same code in both cases?

Reply:
Is not possible to fallthrough in a type switch